BrainPortal/app/controllers/data_providers_controller.rb

```diff
@@ -76,7 +76,7 @@ def show  #:nodoc:
   end
 
   def new #:nodoc:
-    provider_group_id = ( current_project && current_project.id ) || current_user.own_group.id
+    provider_group_id = default_target_group_id || current_user.own_group.id
```

Comment:
Why leave the `|| current_user.own_group.id` here given the logic is in the method `default_target_group_id` ?